contrib/debian/changelog

```diff
@@ -1,3 +1,9 @@
+zcash (4.7.0~rc1) stable; urgency=medium
```

Comment:
```suggestion
zcash (4.7.0-rc1) stable; urgency=medium
```

Reply:
This is how the Debian tooling generates it; I'm not messing with their semantics.